.github/workflows/ci.yml

```diff
@@ -35,11 +35,23 @@ jobs:
           lcov --rc lcov_branch_coverage=1 --remove build/coverage.info '*mocks*' --output-file build/coverage.info
           lcov --rc lcov_branch_coverage=1 --list build/coverage.info
       - name: Check Coverage
-        uses: ChicagoFlutter/lcov-cop@v1.0.2
-        with:
-          path: "build/coverage.info"
-          min_coverage: 100
-          exclude: "**/*test*"
+        env:
+          MIN_COVERAGE: 100
```

Comment:
Can you print the minimum coverage value as 
`echo "Required line and branch coverages:    $MIN_COVERAGE"`

Reply:
ok